apps/src/sites/code.org/pages/public/yourschool.js

```diff
@@ -33,9 +33,15 @@ function showYourSchool() {
         alertText={$('#your-school').data("parameters-alert-text")}
         alertUrl={$('#your-school').data("parameters-alert-url")}
         hideMap={$('#your-school').data("parameters-hide-map")}
+        fusionTableId={$('#your-school').data("parameters-table-id")}
```

Comment:
Nit: jQuery inefficiency.  You should only do the element lookup once when possible.
```js
const yourSchool = $('#your-school');
const rawSchoolId = yourSchool.data("parameters-schoolId");
const rawSchoolZip = yourSchool.data("parameters-schoolZip");
// etc...
```